apps/studio/components/interfaces/GraphQL/GraphiQL.tsx

```diff
@@ -303,6 +303,22 @@ export const GraphiQLInterface = ({ theme }: GraphiQLInterfaceProps) => {
                         Headers
                       </UnStyledButton>
 
+                      {canReadJWTSecret && roleImpersonationEnabledFlag && (
+                        <UnStyledButton
```

Comment:
could add text-sm to the unstyled buttons? the text seems slightly bigger than the rest of the dashboard

Reply:
Yep, styling is from the default GraphiQL styles. Updated!